src/net_components/layers/conv2d.jl

```diff
@@ -2,11 +2,15 @@ using JuMP
 using ConditionalJuMP
 
 export Conv2d
-export Padding, same, valid
+export Padding, SamePadding, ValidPadding
 
-@enum DynamicPadding same=1 valid=2
-FixedPadding = Union{Int64, Tuple{Int64, Int64}, Tuple{Int64, Int64, Int64, Int64}}
-Padding = Union{DynamicPadding, FixedPadding}
+struct SamePadding end
+Base.show(io::IO, p::SamePadding) = print(io, "same")
```

Comment:
This ensures that it still displays as `same` when we are printing out the `Conv2d` layer:

https://github.com/samuelemarro/MIPVerify.jl/blob/709571d8f90213e99bff30b8610ba1167885fbe6/test/net_components/layers/conv2d.jl#L108